core/build.sbt

```diff
@@ -0,0 +1,68 @@
+import Dependencies._
+
+scalariformSettings
+
+name := "pureconfig"
+
+organization := "com.github.melrief"
+
+version := "0.4.0"
```

Comment:
Should the version be externalized from any one project so that it's less tedious to keep all sub-projects uniform?
Perhaps it ought to be in a `version.sbt` at the root.